src/components/App/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import logo from '../../logo.svg';
+import './App.css';
+import MainApp from '../MainApp';
+import {
+    useSelector,
+} from 'react-redux';
+
+function App() {
+    const todos = useSelector((state: {list: { todos: any[] }}) => state.list.todos);
+  return (
+      // туду лист для юзеров:
+    <div className="App main">
+      <header className="App-header">
+        TODO list with users:
+        {/*<img src={logo} className="App-logo" alt="logo" />*/}
```

Comment:
I would remove this from the markup, and if really necessary I would place it above the `return`:
```
// TODO: include <img /> with logo inside te markup